test/e2e/integration/utils.go

```diff
@@ -62,8 +69,12 @@ func (cmd *Cmd) RunCmdWithFlags() error {
 				break
 			}
 			if len(line) > 0 {
-				logWriter.WriteString(fmt.Sprintf("%s\n", line))
-				logWriter.Flush()
+				if _, err1 := logWriter.WriteString(fmt.Sprintf("%s\n", line)); err1 != nil {
+					cmd.Logger.Errorf("failed to write to log file for command %s: %v", cmd.Task, err1)
+				}
+				if err1 := logWriter.Flush(); err1 != nil {
+					cmd.Logger.Errorf("failed to flush log file for command %s: %v", cmd.Task, err1)
+				}
```

Comment:
here as well, you can use `err` variable name as it's inside the if block, it won't cause an issue.